Source/RP0/Avionics/ModuleProceduralAvionics.cs

```diff
@@ -676,7 +677,8 @@ internal static float GetStatsForTechNode(ProceduralAvionicsTechNode techNode, f
             else if (controllableMass <= 0)
                 controllableMass = techNode.interplanetary ? 0.5f : 100f;
 
-            massKG = GetAvionicsMass(techNode, controllableMass) * 1000;
+            var avionicsMass = GetAvionicsMass(techNode, controllableMass);
```

Comment:
Should not omit type if it isn't apparent from the right side of the assignment.